### DISCUSSION TITLE
Streaming support

### DISCUSSION THREAD

**Discussion by jullle3** [projectdiscovery/subfinder#836] [Q&A] [answered]
Hi.

Is streaming of subfinder results supported, if yes from which versions? 

I tried looking around and found no answer

Thank you very much.

**Comment by olearycrew** [comment]
Hi @jullle3 - subfinder does stream the output to `stdout` - is there some other use case you're looking to support?

**Reply by jullle3** [reply]
@olearycrew Hi, yea sorry for being unprecise. Lets say subfinder scan on apple.com takes 30 seconds, does it stream the results to stdout during the 30 sec or does it output it all at once when the execution has completed?

**Reply by olearycrew** [reply]
I believe that it streams it as it finds each one.

**Reply by jullle3** [reply]
@olearycrew Hmm. In that case i'll add an issue since the documentation IMO clearly should state this. 

Actually i believe the opposite, it only outputs when it has found every subdomain, but i might be wrong. 

I'll open an issue for clarification

**Comment by jullle3** [comment, marked as answer]
Subfinder does not stream output during enumeration, see a developers answer here:
![image](https://github.com/projectdiscovery/subfinder/assets/22666834/fb0435ed-edcd-4a88-adee-7f85ebf9b6c2)




@olearycrew Tagging you for clarification purposes
